eng/ci/templates/official/jobs/run-non-e2e-tests.yml

```diff
@@ -43,6 +43,28 @@ jobs:
       keyVaultName: azure-functions-host-$(LeaseBlob)
       secretsFilter: '*'
 
+  - task: AzureCLI@2
+    displayName: 'Azure CLI '
+    inputs:
+      azureSubscription: Azure-Functions-Host-CI-internal
+      addSpnToEnvironment: true
+      scriptType: bash
+      scriptLocation: inlineScript
+      inlineScript: |
+        echo "##vso[task.setvariable variable=ARM_CLIENT_ID]$servicePrincipalId" 
+        echo "##vso[task.setvariable variable=ARM_ID_TOKEN]$idToken"
+        echo "##vso[task.setvariable variable=ARM_TENANT_ID]$tenantId"
+
+  - bash: |
```

Comment:
I am assuming this step is here to make the az login available to the tests later in the pipeline.

Can you add a comment explaining as such? Highlighting how this login context is different than the `AzureCLI` task above.

Reply:
Correct. Will add a comment.